### PR TITLE
FEATURE: Landing Page adjustment

### PR DESCRIPTION
**Pull request by pedrorcruzz**
## What was done?

- Changed  link of github icon on LP

### CODE REVIEW

File: pages/landing/Footer.tsx [hunk 1]
```diff
@@ -33,9 +33,9 @@ const Footer = () => {
           </div>
           {/* Social Brands */}
           <div className="mt-3 space-x-2">
-            <a
+            <Link
               className="inline-flex h-10 w-10 items-center justify-center rounded-full text-center text-gray-500 transition hover:bg-gray-100 focus:outline-none focus:ring-2 focus:ring-blue-600 focus:ring-offset-2 focus:ring-offset-white dark:text-gray-500 dark:hover:bg-gray-800 dark:hover:text-gray-200"
-              href="#"
+              href="https://github.com/danielcorreia-dev/edu-ai"
```
Comment (dancorreia-swe) [inline pages/landing/Footer.tsx:38]: You should put a redirect also put a _blank redirect on the Link component, try adding this: 
```suggestion
              href="https://github.com/danielcorreia-dev/edu-ai"
              target="_blank"
```

**Review by dancorreia-swe — CHANGES_REQUESTED** [review]
Just a small adjustment, great contribution! ;D

File: pages/landing/Footer.tsx (hunk 1 above)
Comment (pedrorcruzz) [reply pages/landing/Footer.tsx:38]: Consertei.

**Review by dancorreia-swe — APPROVED** [review]
Everything just on the spot! Nice

**Pull request merged**